packages/web-components/src/components/va-text-input/va-text-input.tsx

```diff
@@ -138,7 +138,7 @@ export class VaTextInput {
   /**
    * Whether or not the component will use USWDS v3 styling.
    */
-  @Prop() uswds?: boolean = false;
+  @Prop({reflect: true}) uswds?: boolean = false;
```

Comment:
This is needed for the component to refresh the itself with the uswds css when it's used inside another component.